src/components/main/friends/sidebar/mod.rs

```diff
@@ -166,7 +166,7 @@ pub fn FindFriends(cx: Scope, account: Account, add_error: UseState<String>) ->
                     remote_friend.set(evt.value.clone());
                 },
                 on_enter: move |_| {
-                    let did = DID::try_from(String::from("did:key:") + &(remote_friend.clone().to_string()));    
+                    let did = DID::try_from(String::from("did:key:") + &(remote_friend.clone().to_string()));   
```

Comment:
small change, should we use the`format!("did:key:{}", remote_friend.clone())` macro instead?

Reply:
Updated